devops/demo/landing-page/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# sha256 as of 2022-02-16
+FROM nginx:mainline-alpine@sha256:050385609d832fae11b007fbbfba77d0bba12bf72bc0dca0ac03e09b1998580f
```

Comment:
Is there a reason we're using alpine here instead of a Debian/Ubuntu base like we do for everything else?

Reply:
Only that we are using an Alpine base for the other nginx containers in the k8s environment (docs and HTTPS Everywhere).

Reply:
Ack, makes sense then.